srcpkgs/etlegacy/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'etlegacy'
+pkgname=etlegacy
+version=2.77.1
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release  -DCMAKE_LIBRARY_PATH=/usr \
+ -DCMAKE_INSTALL_PREFIX=/usr -DINSTALL_DEFAULT_BASEDIR=/usr/lib/etlegacy \
+ -DCLIENT_GLVND=1 -DBUNDLED_LIBS=0 -DFEATURE_AUTOUPDATE=0 \
+ -DINSTALL_OMNIBOT=0"
+hostmakedepends="libtool pkg-config"
+makedepends="libglvnd-devel glew-devel SDL2-devel zlib-devel minizip-devel
+ libjpeg-turbo-devel libcurl-devel lua54-devel libvorbis-devel libtheora-devel freetype-devel
+ libpng-devel sqlite-devel libopenal-devel"
+depends="curl"
+short_desc="Project based on Wolfenstein: Enemy Territory"
+maintainer="biopsin <biopsin@teknik.io>"
+license="GPL-3.0-or-later"
+homepage="https://www.etlegacy.com"
+changelog="https://github.com/etlegacy/etlegacy/wiki/Changelog"
+distfiles="https://github.com/etlegacy/etlegacy/archive/refs/tags/v${version}.tar.gz"
+checksum=730a8a52435884b922d8c280fcdeace648902399798c7a973da72fbc4163ebe2
+
+case "${XBPS_TARGET_MACHINE}" in
+		x86_64*) configure_args+=" -DCROSS_COMPILE32=0";;
+		i686*) configure_args+=" -DCROSS_COMPILE32=1";;
+		*) broken="This architecture is not currently supported";;
```

Comment:
Build files suggest that cross compiling for arm is an option.

Reply:
this whole block just looks wrong

the purpose of CROSS_COMPILE32 as far as I can tell is to force a 32-bit build on a 64-bit x86 host, since x86_64 game is not compatible with 32-bit x86 binary mods

I also don't see anything blocking function on other CPU archs

Reply:
Ive added the blocking function, just strugling wrapping my head around the cross compile block.
I'll ponder some more, thank you for the comment.

Reply:
@q66 mind eyeball this block once more before push..
This eliminates the "--Forcing i386 to cross compile 32bit" build i686 on x86_64 host, 
however I have'nt tested other way around.
```
archs="i686* x86_64*"
if [ -z "$CROSS_BUILD" ] && [ "$XBPS_WORDSIZE" = 32 ]; then
	configure_args+=" -DCROSS_COMPILE32=1"
else
	configure_args+=" -DCROSS_COMPILE32=0"
fi
```